test/acceptance/post_test.exs

```diff
@@ -73,8 +73,8 @@ defmodule Test.Acceptance.PostTest do
     end
 
     attributes do
-      attribute(:id, :uuid, primary_key?: true)
-      attribute(:name, :string)
+      attribute(:id, :uuid, primary_key?: true, allow_nil?: false)
```

Comment:
do we need the changes in this file?

Reply:
not exactly, wanted to keep them as reference. will remove if you want that.

Reply:
Its fine to leave it 👍

Reply:
ok